src/statistics.cc

```diff
@@ -0,0 +1,87 @@
+
+/**
+ * DeepDetect
+ * Copyright (c) 2020 Jolibrain
+ * Author: Mehdi Abaakouk <mehdi.abaakouk@jolibrain.com>
+ *
+ * This file is part of deepdetect.
+ *
+ * deepdetect is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * deepdetect is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with deepdetect.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <chrono>
+
+#include "apidata.h"
+#include "statistics.h"
```

Comment:
The file could be named `service_statistics.h` or `service_stats.h`. Point being that within an ML software, `statistics.h` may well handle statistical computations / algorithms in a more general sense. Also, the class itself is `ServiceStatistics` so I believe it'd make even more sense to rename.

Reply:
Yes, renamed.